phpstan-baseline.neon

```diff
@@ -3,3 +3,14 @@ parameters:
         - # https://github.com/phpstan/phpstan-strict-rules/issues/130
             message: '#^Call to static method PHPUnit\\Framework\\Assert::.* will always evaluate to true\.$#'
             path: tests/
+
+        - # Symfony 5.4 deprecations
+            message: "#^Call to method import\\(\\) on an unknown class Symfony\\\\Component\\\\Routing\\\\RouteCollectionBuilder\\.$#"
+            count: 2
+            path: tests/App/AppKernel.php
+
+        -
```

Comment:
Those two can be fixed same way they were fixed on media bundle.

Reply:
I want to move this to the mentioned PR.

Reply:
Check https://github.com/sonata-project/SonataClassificationBundle/pull/787

Reply:
Thanks @jordisala1991 :)